packages/databricks-vscode/src/configuration/ConnectionCommands.ts

```diff
@@ -119,7 +119,7 @@ export class ConnectionCommands implements Disposable {
     }
 
     attachClusterQuickPickCommand() {
-        return async (title?: string) => {
+        return async () => {
```

Comment:
We still use the title in the `EnvironmentDependenciesVerifier->checkCluster`. We can remove the title from there too, but might as well leave it. Or is the `title` check here is not good enough and you are fixing a bug here by removing the argument?

Reply:
I messed up when fixing merge conflicts.